src/controller/python/chip/yaml/runner.py

```diff
@@ -143,9 +143,9 @@ def __init__(self, test_step, cluster: str, context: _ExecutionContext):
           'cluster': Name of cluster which to invoke action is targeting.
           'context': Contains test-wide common objects such as DataModelLookup instance.
         Raises:
-          ParsingError: Raised if there is a benign error, and there is currently no
+          ActionCreationError: Raised if there is a benign error, and there is currently no
             action to perform for this write attribute.
```

Comment:
Pre-existing, but what does "write attribute" have to do with InvokeAction?

Reply:
Good catch, this was a copy paste error for user. I have updated the comment now to be a lot more descriptive since this is really one of the only few places were we might raise this error